java/api/src/main/java/org/ray/api/options/CallOptions.java

```diff
@@ -18,14 +18,23 @@ private CallOptions(Map<String, Double> resources) {
   public static class Builder {
 
     private Map<String, Double> resources = new HashMap<>();
+    private boolean useDirectCall = DEFAULT_USE_DIRECT_CALL;
 
     public Builder setResources(Map<String, Double> resources) {
       this.resources = resources;
       return this;
     }
 
+    // Since direct call is not fully supported yet (see issue #5559),
+    // users are not allowed to set the option to true.
+    // TODO (kfstorm): uncomment when direct call is ready.
+    // public Builder setUseDirectCall(boolean useDirectCall) {
+    //   this.useDirectCall = useDirectCall;
+    //   return this;
+    // }
```

Comment:
Are the issues in #5599 fixed now?

The first one is not specific to java. Not sure about if the last one still exists though.

Reply:
I will create another PR to restore the skipped test cases mentioned in #5559 and uncomment the `setUseDirectCall` methods in builders.